src/Faction/ui/Info.tsx

```diff
@@ -33,9 +34,10 @@ const useStyles = makeStyles()({
 function DefaultAssignment(): React.ReactElement {
   return (
     <Typography>
-      Perform work/carry out assignments for your faction to help further its cause! By doing so you will earn
-      reputation for your faction. You will also gain reputation passively over time, although at a very slow rate.
-      Earning reputation will allow you to purchase Augmentations through this faction, which are powerful upgrades that
+      Perform work/carry out assignments for your faction to help further its cause! By doing so, you will earn
+      reputation for your faction.&nbsp;
+      {Player.bitNodeN !== 2 && <>You will also gain reputation passively over time, although at a very slow rate. </>}
```

Comment:
I don't like the documentation secretly changing depending on what BN you're in. However, it also shouldn't spoil.
What about adding a conditional based on knowsAboutBitverse that adds a note that the passive rep gain doesn't happen in BN2?